app/routes/_index.tsx

```diff
@@ -8,6 +8,7 @@ import { TypografiTyper } from '~/typer/typografi';
 import SamtykkePanel from '~/komponenter/SamtykkePanel';
 import { useTekster } from '~/hooks/contextHooks';
 import { Språkvelger } from '~/komponenter/språkvelger/språkvelger';
+import { IForsideTekstinnhold } from '~/typer/sanity/sanityForside';
```

Comment:
Skal vi lage et "teknisk gjeld" kort, hvor vi tar en ryddesjau og fjerner alle I-prefixer for Interfaces og E-prefikser forran enums? 

Disse prefixene tilfører ikke verdi men gjør navnene vanskeligere å lese etter min mening. Jeg lener meg også på Typescript sine guidelines som sier at man ikke bør prefixse interfaces og enums:
https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript

Reply:
Nytt kort for å fikse `E` eller `I` prefixes [her](https://trello.com/c/sI0Tg2yj/74-rett-opp-i-teknisk-gjeld-n%C3%A5r-det-kommer-til-e-og-i)